doc/devdocs/readme.md

```diff
@@ -50,8 +50,23 @@ Various tools used by PowerToys. Includes the Visual Studio 2019 project templat
 
 ### Build Prerequisites
 
+1. install the following:
 - Windows 10 1803 (build 10.0.17134.0) or above to build and run PowerToys.
-- Visual Studio 2019 Community edition or higher, with the 'Desktop Development with C++' component and the Windows 10 SDK version 10.0.18362.0 or higher.
+- Visual Studio 2019 Community edition or higher
+
+2. Run this command in cmd/terminal/ps to install all the workloads and components for VS
```

Comment:
This won't actually work in PowerShell (e.g. you'd have to replace %ProgramFiles(x86)% with ${env:ProgramFiles(x86)}. So perhaps just say Command Prompt. Developers can translate.

Reply:
adjusted